CollectionvViewDemo/AppDelegate/AppDelegate.swift

```diff
@@ -0,0 +1,33 @@
+//
+//  AppDelegate.swift
+//  CollectionvViewDemo
+//
+//  Created by Bence Pattogato on 23/08/2021.
+//
+
+import UIKit
+
+@main
+class AppDelegate: UIResponder, UIApplicationDelegate {
+    func application(_ application: UIApplication, didFinishLaunchingWithOptions launchOptions: [UIApplication.LaunchOptionsKey: Any]?) -> Bool {
+        // Override point for customization after application launch.
+        return true
+    }
+
+    // MARK: UISceneSession Lifecycle
+
+    func application(_ application: UIApplication, configurationForConnecting connectingSceneSession: UISceneSession, options: UIScene.ConnectionOptions) -> UISceneConfiguration {
+        // Called when a new scene session is being created.
+        // Use this method to select a configuration to create the new scene with.
+        return UISceneConfiguration(name: "Default Configuration", sessionRole: connectingSceneSession.role)
+    }
+
+    func application(_ application: UIApplication, didDiscardSceneSessions sceneSessions: Set<UISceneSession>) {
+        // Called when the user discards a scene session.
+        // If any sessions were discarded while the application was not running, this will be called shortly after application:didFinishLaunchingWithOptions.
+        // Use this method to release any resources that were specific to the discarded scenes, as they will not return.
+    }
```

Comment:
[nit] Since it's a showcase app, I _think_ it's best to include as little code as absolutely necessary, so we reduce the cognitive load and the number of lines that the readers might just waste time with. So I'd probably remove all unused code.
Probably these are mandatory methods of the `UIApplicationDelegate`, but removing the the comments and leaving a function with no body at all might help understand that there is nothing to see here.